### PR TITLE
rename from aldryn-client to divio-cli

### PR DESCRIPTION
**Pull request by Chive**
- [x] Create new PyPI release
- [ ] Rename repo and move to @divio

### CODE REVIEW

File: scripts/entrypoint.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import re
 import sys
 
-from aldryn_client.cli import cli
```
Comment (stefanfoulis) [inline scripts/entrypoint.py]: much ``cli`` :-)

**Comment by Chive** [comment]
@stefanfoulis yup I did - the new additions to the `version` command gives us some insight in where the CLI is actually installed. I hope this to be useful when debugging with beginner users.

## As Python package

```bash
(.venv) ❯ divio version
divio-cli 2.3.6a1 from /path/divio-cli/.venv/bin

You have the latest version of divio-cli!
```

## As binary

```bash
❯ divio version
divio-cli 2.3.6a1 from /usr/local/bin

You have the latest version of divio-cli!
```

**Comment by GaretJax** [comment]
@stefanfoulis sys.executable actually gives you the location of the python interpreter. @Chive it should be ok for your needs.

**Comment by Chive** [comment]
@GaretJax exactly - which in case of a python package would give us the folder where the script "should" be installed - and if it's a binary, it would be the location of that binary since the interpreter is bundled

**Pull request merged**